<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="UsuarioColsControllerTest" tests="7" time="0.102752723" failures="6" errors="1" skipped="0" assertions="6" timestamp="2014-12-02T21:58:28-05:00">
  <testcase name="test_should_create_usuario_col" time="0.833661259" assertions="1">
    <failure type="Minitest::Assertion" message="&quot;UsuarioCol.count&quot; didn't change by 1....">
"UsuarioCol.count" didn't change by 1.
Expected: 3
  Actual: 2 (Minitest::Assertion)
/home/gustavo/repos/rails/modulo_seguimiento/implementacion/test/controllers/usuario_cols_controller_test.rb:21    </failure>
  </testcase>
  <testcase name="test_should_destroy_usuario_col" time="0.015743688" assertions="1">
    <failure type="Minitest::Assertion" message="&quot;UsuarioCol.count&quot; didn't change by -1....">
"UsuarioCol.count" didn't change by -1.
Expected: 1
  Actual: 2 (Minitest::Assertion)
/home/gustavo/repos/rails/modulo_seguimiento/implementacion/test/controllers/usuario_cols_controller_test.rb:44    </failure>
  </testcase>
  <testcase name="test_should_get_edit" time="0.01075556" assertions="1">
    <failure type="Minitest::Assertion" message="Expected response to be a &lt;success&gt;, but was &lt;302&gt;">
Expected response to be a &lt;success&gt;, but was &lt;302&gt; (Minitest::Assertion)
/home/gustavo/repos/rails/modulo_seguimiento/implementacion/test/controllers/usuario_cols_controller_test.rb:35    </failure>
  </testcase>
  <testcase name="test_should_get_index" time="0.010623557" assertions="1">
    <failure type="Minitest::Assertion" message="Expected response to be a &lt;success&gt;, but was &lt;302&gt;">
Expected response to be a &lt;success&gt;, but was &lt;302&gt; (Minitest::Assertion)
/home/gustavo/repos/rails/modulo_seguimiento/implementacion/test/controllers/usuario_cols_controller_test.rb:11    </failure>
  </testcase>
  <testcase name="test_should_get_new" time="0.007918044" assertions="1">
    <failure type="Minitest::Assertion" message="Expected response to be a &lt;success&gt;, but was &lt;302&gt;">
Expected response to be a &lt;success&gt;, but was &lt;302&gt; (Minitest::Assertion)
/home/gustavo/repos/rails/modulo_seguimiento/implementacion/test/controllers/usuario_cols_controller_test.rb:17    </failure>
  </testcase>
  <testcase name="test_should_show_usuario_col" time="0.005337694" assertions="1">
    <failure type="Minitest::Assertion" message="Expected response to be a &lt;success&gt;, but was &lt;302&gt;">
Expected response to be a &lt;success&gt;, but was &lt;302&gt; (Minitest::Assertion)
/home/gustavo/repos/rails/modulo_seguimiento/implementacion/test/controllers/usuario_cols_controller_test.rb:30    </failure>
  </testcase>
  <testcase name="test_should_update_usuario_col" time="0.006242131" assertions="0">
    <error type="ActionController::UrlGenerationError" message="ActionController::UrlGenerationError: No route matches {:action=&gt;&quot;show&quot;, :controller=&gt;&quot;usuario_cols&quot;, :id=&gt;nil} missing required keys: [:id]...">
ActionController::UrlGenerationError: No route matches {:action=&gt;"show", :controller=&gt;"usuario_cols", :id=&gt;nil} missing required keys: [:id]
    test/controllers/usuario_cols_controller_test.rb:40:in `block in &lt;class:UsuarioColsControllerTest&gt;' (ActionController::UrlGenerationError)
/home/gustavo/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/actionpack-4.1.6/lib/action_dispatch/routing/route_set.rb:192    </error>
  </testcase>
</testsuite>
